templates/layouts/base.html

```diff
@@ -41,6 +41,8 @@
     {% set show_nav_toggle = true %}
   {% endif %}
 {% endif %}
+{% set newline = "
```

Comment:
What is the point of this line? (Non-blocking)

Reply:
This is part of the fix for the weird indentation in the code blocks.

As far as I can tell, a literal newline is the only way to get a newline into a string in Zola's templates. Previously, we put that literal newline in the code block generation code. But because the new CI is forcing us to indent that code, we're getting some spaces along with out newline.

So I set up this `newline` variable so I could get the newline in with string concatenation.

Reply:
See also: https://github.com/bevyengine/bevy-website/pull/951